src/powerpwn/cli/arguments.py

```diff
@@ -227,6 +227,19 @@ def copilot_studio_modules(parser: argparse.ArgumentParser, module: str):
         parser.add_argument("-t", "--timeout", help="The timeout for the enumeration process to have (in seconds)", default=300)
 
 
+def module_powerpages(parser: argparse.ArgumentParser):
+    powerpages = parser.add_parser(
+        "powerpages",
+        description="Test a power pages API anonymous access vulnerability",
+        help="Test a power pages API anonymous access vulnerability",
+    )
+    powerpages.add_argument(
+        "-url",
+        help="The url of the power pages domain to be tested\nFormat the url as such: 'https://<your_domain>.powerappsportals.com'",
```

Comment:
Not critical but _argparser_ seems to disregard `\n`, can we use tripe-quoted strings or just add a full-stop instead of a new line?

Reply:
done